src/content/topics/integrations/code-references/index.mdx

```diff
@@ -132,6 +132,6 @@ To learn more about `.ldignore` files, read the [`ld-find-code-refs` documentati
 
 The code references feature scans your source code for occurrences of your flag keys. This process requires your source code to reference flag keys exactly as they appear in LaunchDarkly.
 
-However, by default, the React SDK camel-cases all flag keys for easier access with dot notation. To use the code references feature in conjunction with the React SDK, configure the SDK to disable this camel-casing feature.
+However, by default, the React SDK camel-cases all flag keys for easier access with dot notation. To use the code references feature in conjunction with the React SDK, enable camel-case alias generation or configure the SDK to disable this camel-casing feature.
```

Comment:
I believe you are referring to a Code References feature, rather than an SDK feature, when you mention "camel-case alias generation". Is this correct? Can this link to the appropriate code references doc/section?

Right now this text change is ambiguous as to what it refers.

Reply:
This is correct. I'll add a link and will expand about the CR feature.